### DISCUSSION TITLE
Problems with display of a simple point file from QGIS in QField

### DISCUSSION THREAD

**Discussion by jamssiwater** [opengisch/QField#4351] [Q&A]
I have set up a working QField project transferring map data from a smallish (OSX) QGIS project.  One particular project point file does not appear as a layer in QField (though the named point file is visible in the list of imported QField files).  The problematic file was imported into QGIS using delimited text format (.csv), and displays fine in QGIS..., No error messages in QField!

Grateful for any pointers.
J

**Comment by nirvn** [comment]
@jamssiwater , can you share a test project showing this missing layer?

**Comment by jamssiwater** [comment]
Not sure how to do that (you mean in QField?), happy to try 
J

**Reply by jamssiwater** [reply]
resolved this one via a reimport, though ideally the actually cloud file would be "pulled" live from a Postgres routine
